src/main/java/at/fh/joanneum/irfc/service/VoteService.java

```diff
@@ -0,0 +1,86 @@
+package at.fh.joanneum.irfc.service;
+
+import at.fh.joanneum.irfc.model.vote.VoteDTO;
+import at.fh.joanneum.irfc.model.vote.VoteMapper;
+import at.fh.joanneum.irfc.persistence.entiy.VoteEntity;
+import at.fh.joanneum.irfc.persistence.repository.VoteRepository;
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static java.util.Objects.isNull;
+
+/**
+ * @author dominik.jaeger@edu.fh-joanneum.at
+ **/
+@RequestScoped
+public class VoteService {
+
+    @Inject
+    VoteRepository voteRepository;
+
+
+    public List<VoteDTO> getAll() {
```

Comment:
It's not wrong but we only need the calls from the puml (green public, red private). So you don't have to do it if you don't want to. No need to remove this its jfyi

Reply:
already knew that when implementing, but thought if we want to add some functionalities it would be already there and we wouldn't have to implement it later